man/dcfldd.1

```diff
@@ -207,6 +207,11 @@ verifylog:=COMMAND
 Exec and write verify results to process COMMAND.
 .TP
 .B
+diffwr=[on|off]
+Write only to output if destination block content is differs. This operation 
+applies to any of=FILE that follows (diffwr= must be put before of=)
+.TP
+.B
```

Comment:
There is no need to change this file, as it will be automatically generated during build. Changes should go only to `man/dcfldd.txt`.